### DISCUSSION TITLE
Secure boot for HPM6280

### DISCUSSION THREAD

**Discussion by sadkotheguest** [hpmicro/hpm_sdk#25] [General]
According to the 19.4 安全启动 (Secure Boot) of HPM6200_UM_V1_0 the HPM6280 MCU has the capability of using SecureBoot. 
Do you have any guide or example how to prepare the firmware image and flash it to make the Secure Boot work?

**Comment by helloeagleyang** [comment]
Hi,  Please download the tool via the below link:
https://github.com/hpmicro/hpm_manufacturing_tool/releases/download/v0.3.1/HPMicro_Manufacturing_Tool_v0.3.1.zip 

It provides one-stop image editor. It is easy to generate certificates, sign image and encrypt image using the image editor.
